src/Tensors/dense.jl

```diff
@@ -626,6 +626,9 @@ function permute_reshape(T::DenseTensor{ElT,NT,IndsT},
   return reshape(T,newinds)
 end
 
+Base.conj(T::DenseTensor{<:Real,N}) where {N} = T
+Base.conj(T::DenseTensor) = Tensor(Dense(conj(store(T))), copy(inds(T)))
```

Comment:
It seems like we could have a more generic definition:
```julia
Base.conj(T::Tensor) = Tensor(conj(store(T)), copy(inds(T)))
```
and then every storage type must define there own version of `conj`. Each version should return a storage object of the same type, so the `Dense(...)` constructor should not be necessary.

Reply:
And just as an FYI, for generic function like that (where it seems like there is a reasonable definition for Tensors of most storage types), I have been trying to put them in the file `src/Tensors/tensor.jl`.

Reply:
Indeed this could be done. I thought about it but was not sure whether you want this fix to be applied also for the block sparse tensors. But I guess it's not a big deal to change it afterwards in any case. I will update the PR.

Reply:
Yes it's no big deal to change it, and we can always overload the general definition for other Tensor/storage types.